docs/perf_client.rst

```diff
@@ -216,6 +217,136 @@ tell perf\_client to send batch-size 4 requests of shape [ 3, 224, 224 ]::
 
   $ perf_client -m mymodel -b 4 --shape IMAGE:3,224,224
 
+Real Input Data
+^^^^^^^^^^^^^^^
+
+The performance of some of the models is highly dependent on the data used.
+For such cases users can provide data to be used with every inference request
+made by client in a json file. The client will keep cycling through the data
+to maintain steady inferences.
+
+Every data object should specify data for all the inputs with exact size
+as expected by the model. Otherwise, an error will be thrown. The following
+example describes data for a model with inputs named, INPUT0 and INPUT1,
+shape [16] and data type STRING: ::
+
+
+  {
+    "count" : 0,
+    "data" :
+     [
+        {
+          "INPUT0" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        },
+        {
+          "INPUT0" : ["2", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        },
+        {
+          "INPUT0" : ["3", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        },
+        {
+          "INPUT0" : ["4", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        }
+        .
+        .
+        .
+      ]
+  }
+
+
+The "count" field describes the number of data objects to use from the file. A specification of zero
```

Comment:
This is a strange use of count. I would have it be the number of entries in data (which is redundant) or just drop it.

Reply:
Dropped the count specification